core/server/data/migrations/versions/4.46/2022-04-20-08-39-map-subscribers-to-default-newsletter.js

```diff
@@ -0,0 +1,74 @@
+const chunk = require('lodash/chunk');
+const logging = require('@tryghost/logging');
+const ObjectID = require('bson-objectid');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Adding existing subscribers to default newsletter');
+
+        const newsletter = await knex('newsletters')
+            .orderBy('sort_order', 'asc')
+            .first('id', 'name');
+
+        if (!newsletter) {
+            logging.info(`Default newsletter not found - skipping`);
+            return;
+        }
+
+        // This is at the start of the up() instead of at the end of the down()
+        // to maintain idempotency
+        logging.info('Removing existing newsletter subscriptions');
+        await knex('members_newsletters').delete();
+
+        logging.info(`Subscribing members to newsletter '${newsletter.name}'`);
+
+        const memberIds = await knex('members')
+            .where({subscribed: true})
+            .pluck('id');
+
+        if (!memberIds.length) {
+            logging.info(`No members to subscribe - skipping`);
+            return;
+        }
+
+        logging.info(`Found ${memberIds.length} members to subscribe`);
+
+        let pivotRows = [];
+        // eslint-disable-next-line no-restricted-syntax
+        for (const memberId of memberIds) {
```

Comment:
Could you switch to `const pivotRows = membersIds.map(m => ...` here to save the for-loop and therefore the eslint exemption?